### PR TITLE
Mainline

### PR DESCRIPTION
**Pull request by hasskell**
Create a basic repository for accessing a neo4j, the main controller that lets the user enter some values in the form and search for flights from origin to destination.

### CODE REVIEW

File: src/main/java/com/dlobanovskis/flightplanner/forms/Journey.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * Created by: Dmitrijs Lobanovskis
+ * Email: dlobanovskis@gmail.com
+ * Date: 18/01/2018
+ * License:
+ *
+ * This is a journey class contains user entered information
+ */
+
+@Data
+public class Journey {
```
Comment (IvanFox) [inline src/main/java/com/dlobanovskis/flightplanner/forms/Journey.java]: Can any of the following arguments be null?

File: src/main/java/com/dlobanovskis/flightplanner/controller/Controller.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ * Created by: Dmitrijs Lobanovskis
+ * Email: dlobanovskis@gmail.com
+ * Date: 18/01/2018
+ * License:
+ *
+ * Main controller class, user construct his journey here
+ */
+
+@org.springframework.stereotype.Controller
+public class Controller {
```
Comment (IvanFox) [inline src/main/java/com/dlobanovskis/flightplanner/controller/Controller.java]: Try to be more specific with Controller names. The name here doesnt reflect what it does

File: src/main/java/com/dlobanovskis/flightplanner/controller/Controller.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+    @Autowired
```
Comment (IvanFox) [inline src/main/java/com/dlobanovskis/flightplanner/controller/Controller.java]: Why you prefer setter injection over constructor?

Comment (hasskell) [reply src/main/java/com/dlobanovskis/flightplanner/controller/Controller.java]: Good point didn't know that setter might not be called

File: src/main/java/com/dlobanovskis/flightplanner/forms/Journey.java (hunk 1 above)
Comment (hasskell) [reply src/main/java/com/dlobanovskis/flightplanner/forms/Journey.java]: For now yes class will be a bit different later

**Pull request merged**